vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/model/serializers/BeanSerializerDelegator.java

```diff
@@ -70,11 +71,24 @@ protected BeanSerializerBase withIgnorals(Set<String> toIgnore) {
         return new BeanSerializerDelegator(this, toIgnore);
     }
 
+    @Override
+    protected BeanSerializerBase withByNameInclusion(Set<String> set,
+            Set<String> set1) {
+        return null;
```

Comment:
Replace `null` with `this`  like other methods already implemented

Reply:
or even better throw an exception saying unimplemented method

Reply:
updated with an exception